src/LibraryViewExtensionWebView2/LibraryViewController.cs

```diff
@@ -361,7 +361,7 @@ private void Browser_CoreWebView2InitializationCompleted(object sender, CoreWebV
             var libHTMLURI = "Dynamo.LibraryViewExtensionWebView2.web.library.library.html";
             var stream = LoadResource(libHTMLURI);
 
-            var libMinURI = "Dynamo.LibraryViewExtensionWebView2.web.library.librarie.min.js";
+            var libMinURI = "Dynamo.LibraryViewExtensionWebView2.web.library.build.librarie.min.js";
```

Comment:
If you are uncompressing the tgz file in e.g. `Packages/LibrarieJS` and you want to avoid modifying this path then you can copy the librarie.min.js file to the web/library folder with the copy task:
https://learn.microsoft.com/en-us/visualstudio/msbuild/copy-task?view=vs-2022

Reply:
> If you are uncompressing the tgz file in e.g. `Packages/LibrarieJS` and you want to avoid modifying this path then you can copy the librarie.min.js file to the web/library folder with the copy task: https://learn.microsoft.com/en-us/visualstudio/msbuild/copy-task?view=vs-2022

I think if we wanted to keep the same structure as **NotificationsCenter** make sense to put the bundle under /build